internal/api/attestationconfig/client/client.go

```diff
@@ -121,7 +130,56 @@ func (a Client) DeleteList(ctx context.Context, attestation variant.Variant) err
 	return put(ctx, a.s3Client, a.bucketID, path.Join(constants.CDNAttestationConfigPrefixV1, attestation.String(), "list"), bt)
 }
 
-func (a Client) addVersionToList(ctx context.Context, attestation variant.Variant, fname string) error {
+func (a client) deleteAzureSEVSNVersion(versions attestationconfig.AzureSEVSNPVersionList, versionStr string) (ops []crudOP, err error) {
+	versionStr = versionStr + ".json"
+	variant := variant.AzureSEVSNP{}
+	// delete version file
+	filePath := fmt.Sprintf("%s/%s/%s", constants.CDNAttestationConfigPrefixV1, variant.String(), versionStr)
+	ops = append(ops, deleteCmd{
+		path: filePath,
+	})
+
+	// delete signature file
+	sigPath := fmt.Sprintf("%s/%s/%s", constants.CDNAttestationConfigPrefixV1, variant.String(), versionStr+".sig")
```

Comment:
```suggestion
	filePath := path..Join(constants.CDNAttestationConfigPrefixV1, variant.String(), versionStr)
	ops = append(ops, deleteCmd{
		path: filePath,
	})

	// delete signature file
	sigPath :=path..Join(constants.CDNAttestationConfigPrefixV1, variant.String(), versionStr+".sig")
```
I don't have a strong preference of using `fmt.Sprintf` over `path.Join` (or the other way around), but we should try to keep the functions used to generate the url paths consistent, e.g. [line 159](https://github.com/edgelesssys/constellation/blob/ed8fb1c89cd9f13db110e009e1fbde3d88409378/internal/api/attestationconfig/client/client.go#L159) uses `path.Join` instead.